sockeye/inference.py

```diff
@@ -874,6 +881,29 @@ def __init__(self,
         self.nbest_attention_matrices = nbest_attention_matrices
         self.nbest_scores = nbest_scores
 
+    def json(self, align_threshold: float = 0.0) -> Dict:
+        """
+        Returns a dictionary suitable for json.dumps() representing all
+        the information in the class.
+
+        :param align_threshold: If alignments are defined, only print ones over this threshold.
+        :return: A dictionary.
+        """
+        _d = { 'id': self.sentence_id,
```

Comment:
TranslatorInput calls it `sentence_id`, should be consistent here for people that use the Input/Output objects in an API.